components/google_drive/sources/changes-to-specific-files/changes-to-specific-files.mjs

```diff
@@ -65,14 +47,15 @@ export default {
       // You can pass the same channel ID in watch requests for multiple files, so
       // our channel ID is fixed for this component to simplify the state we have to
       // keep track of.
-      const channelID = this.db.get("channelID") || uuid();
+      const channelID = this._getChannelID() || uuid();
 
       // Subscriptions are keyed on Google's resourceID, "an opaque value that
       // identifies the watched resource". This value is included in request
       // headers, allowing us to look up the watched resource.
-      let subscriptions = this.db.get("subscriptions") || {};
+      let subscriptions = this._getSubscriptions() || {};
 
-      for (const fileID of this.files) {
+      const files = this.files;
+      for (const fileID of files) {
         const {
           expiration,
           resourceId,
```

Comment:
This hook should probably use `this.googleDrive.activateFileHook()` instead of `this.googleDrive.watchFile()`
```
        const {
          expiration,
          resourceId,
        } = await this.googleDrive.activateFileHook(
          channelID,
          this.http.endpoint,
          fileID,
        );
```